test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/util/FormatterTest.scala

```diff
@@ -25,6 +25,7 @@ import java.util._
 class FormatterTest {
   import FormatterTest._
 
+  @noinline
```

Comment:
Why are these necessary with this commit?

Reply:
They're not. I forgot to remove them. Done now.